### DISCUSSION TITLE
How to translate basePath with domain routing and localised pathnames?

### DISCUSSION THREAD

**Discussion by adamccfc** [amannn/next-intl#916] [Q&A]
Hey, I'm using domain routing and localised pathnames and I want to set a basePath in the next config file. 

So in my `middleware.ts` I'm setting the domains like the following: 

```js
  domains: [
    {
      domain: process.env.NEXT_PUBLIC_ENGLISH_DOMAIN,
      defaultLocale: 'en',
      locales: ['en'],
    },
    {
      domain: process.env.NEXT_PUBLIC_WELSH_DOMAIN,
      defaultLocale: 'cy',
      locales: ['cy'],
    },
  ],
```

Then in my `navigation.ts` file I'm setting the pathnames:

```js
export const pathnames = {
  '/': '/',
  '/search': {
    en: '/search',
    cy: '/chwilio',
  },
} satisfies Pathnames<typeof locales>;
```

Then in my `next.config.js` file I'm setting the basePath:

```js
basePath: '/search',
```

The issue I have is when I go to the Welsh domain the basePath does not translate `NEXT_PUBLIC_WELSH_DOMAIN/chwilio` (it 404's) but renders on the English path of `NEXT_PUBLIC_WELSH_DOMAIN/search`. Any help would be greatly appreciated!

Note - I should also mention that `localePrefix` is set to `never`.

**Comment by andeksan** [comment]
Hi, did you solve the issue?
